From 17b0219c1305f56ce2debf0add895db118be24f2 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Thu, 20 Aug 2020 15:51:08 +0800
Subject: [PATCH] Test case for Driver - String.h file

Description: Change the test_len value defined in the MdeModulePkg/Application/UiApp/FrontPage.h.

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPage.c | 2 +-
 MdeModulePkg/Application/UiApp/FrontPage.h | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPage.c b/MdeModulePkg/Application/UiApp/FrontPage.c
index 4b95cccb5c..25fed6daa1 100644
--- a/MdeModulePkg/Application/UiApp/FrontPage.c
+++ b/MdeModulePkg/Application/UiApp/FrontPage.c
@@ -10,7 +10,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include "FrontPage.h"
 #include "FrontPageCustomizedUi.h"
 
-#define MAX_STRING_LEN            200
+#define MAX_STRING_LEN            test_len
 
 EFI_GUID   mFrontPageGuid      = FRONT_PAGE_FORMSET_GUID;
 
diff --git a/MdeModulePkg/Application/UiApp/FrontPage.h b/MdeModulePkg/Application/UiApp/FrontPage.h
index 36207287b9..c84b7fbcc8 100644
--- a/MdeModulePkg/Application/UiApp/FrontPage.h
+++ b/MdeModulePkg/Application/UiApp/FrontPage.h
@@ -21,6 +21,7 @@ extern UINT8  FrontPageVfrBin[];
 extern EFI_FORM_BROWSER2_PROTOCOL      *gFormBrowser2;
 
 
+#define test_len 180 
 #define SMBIOS_TYPE4_CPU_SOCKET_POPULATED BIT6
 
 //
-- 
2.27.0.windows.1

